terraform/modules/monitoring/monitoring.tf

```diff
@@ -0,0 +1,181 @@
+variable "gcp_project" {
+  type = string
+}
+
+variable "gcp_region" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+resource "kubernetes_namespace" "monitor" {
+  metadata {
+    name = "monitor"
+  }
+}
+
+resource "kubernetes_service" "prometheus" {
+  metadata {
+    name      = "prometheus"
+    namespace = "monitor"
+  }
+  spec {
+    selector = {
+      app = kubernetes_pod.prometheus.metadata.0.labels.app
+    }
+    port {
+      port        = 9090
+      target_port = 9090
+    }
+
+    type = "LoadBalancer"
+  }
+}
+
+resource "kubernetes_pod" "prometheus" {
+  metadata {
+    name      = "prometheus"
+    namespace = "monitor"
+    labels = {
+      app = "Prometheus"
+    }
+  }
+
+  spec {
+    service_account_name = kubernetes_service_account.prometheus_account.metadata[0].name
+    container {
+      image = "prom/prometheus:latest"
```

Comment:
Is pulling `latest` safe? Does Prometheus have release channels that provide any kind of stability guarantees?

Reply:
For DockerHub images, `latest` generally means "latest release." And Prometheus releases are pretty stable over time - no big API breaks. I think we're better off staying up-to-date with `latest` than pinning a version. Also, this is one of the recommended install methods: https://prometheus.io/docs/prometheus/latest/installation/

Reply:
Thanks for double checking